src/ocm/components/clusterConfiguration/operators/LvmCheckbox.tsx

```diff
@@ -1,35 +1,43 @@
 import React, { useState } from 'react';
 import { FormGroup, Tooltip } from '@patternfly/react-core';
-import { useFormikContext } from 'formik';
 import {
   ClusterOperatorProps,
-  FeatureSupportLevelBadge,
   getFieldId,
-  OPERATOR_NAME_LVM,
-  operatorLabels,
-  OperatorsValues,
   PopoverIcon,
   useFeatureSupportLevel,
+  operatorLabels,
+  OperatorsValues,
 } from '../../../../common';
 import LvmHostRequirements from './LvmHostRequirements';
-import { getLvmIncompatibleWithCnvReason } from '../../featureSupportLevels/featureStateUtils';
 import { OcmCheckboxField } from '../../ui/OcmFormFields';
 import { useTranslation } from '../../../../common/hooks/use-translation-wrapper';
+import { handleLVMS } from './utils';
+import { useFormikContext } from 'formik';
+import { getLvmIncompatibleWithCnvReason } from '../../featureSupportLevels/featureStateUtils';
 
 const LVM_FIELD_NAME = 'useOdfLogicalVolumeManager';
 
-const LvmLabel = (props: ClusterOperatorProps) => {
+type LvmLabelProps = ClusterOperatorProps; // & { operator: ExposedOperatorName };
+
+const LvmLabel = ({ openshiftVersion, clusterId }: LvmLabelProps) => {
   const { t } = useTranslation();
-  const operatorName = operatorLabels(t)[OPERATOR_NAME_LVM];
+  const [operator, setOperator] = React.useState('');
+
+  const featureSupportLevel = useFeatureSupportLevel();
+  React.useEffect(() => {
+    setOperator(handleLVMS({ openshiftVersion, featureSupportLevel }));
+  }, [featureSupportLevel, openshiftVersion]);
+
+  const operatorName = operatorLabels(t)[operator];
```

Comment:
```suggestion
  const operatorLabel = operatorLabels(t)[operator];
```
as `operator` is in reality an "operator name"